app/src/main/java/com/texthip/thip/ui/group/makeroom/viewmodel/GroupMakeRoomViewModel.kt

```
@@ -2,138 +2,187 @@ package com.texthip.thip.ui.group.makeroom.viewmodel
+                    updateState { it.copy(savedBooks = bookDtos.map { dto -> dto.toBookData() }) }
+                }.onFailure {
+                    updateState { it.copy(savedBooks = emptyList()) }
+                }
+                
+                val groupBooksResult = bookRepository.getBooks("joining")
+                groupBooksResult.onSuccess { bookDtos ->
+                    updateState { it.copy(groupBooks = bookDtos.map { dto -> dto.toBookData() }) }
+                }.onFailure {
+                    updateState { it.copy(groupBooks = emptyList()) }
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)